### PR TITLE
Add Bluetooth service contribution guidelines

### PR DESCRIPTION
**Pull request by noonfom**
This PR adds a `CONTRIBUTING.md` to the `doc` folder.

The document contains the following:
* Description of contribution process
* Description of license and code ownership
* Links to development guides: unit testing, integration testing, external resources
* Mbed OS coding standard link

## Reviewers
@pan- 
@paul-szczepanek-arm 
@AGlass0fMilk

### CODE REVIEW

File: doc/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributing to Mbed OS Experimental BLE Services
+
+Greetings!
+Thank you for expressing an interest in Mbed OS Experimental BLE Services, an incubator for Bluetooth services.
+This is a community-led effort so we welcome all contributions with open arms 😉
```
Comment (paul-szczepanek-arm) [inline doc/CONTRIBUTING.md:5]:
```suggestion
This is a community-led effort so we welcome all contributions with open arms.
```

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## How to contribute
+We would like to make the contribution process as smooth as possible.
+Therefore, we provide an out of the box development environment that can be used with Visual Studio Code.
+Please refer to the main [development guide](./DEVELOPMENT.md) to learn how to setup and use this feature. 
+
+Of course, you are free to use your own development environment. 
+However, please ensure that all dependencies are present in the tree by running the command `bootstrap.sh` in the root repository.
```
Comment (paul-szczepanek-arm) [inline doc/CONTRIBUTING.md:13]: what does that mean?

**Review by paul-szczepanek-arm — APPROVED** [review]
Very concise, I like it.

File: doc/CONTRIBUTING.md (hunk 2 above)
Comment (pan-) [inline doc/CONTRIBUTING.md:13]:
```suggestion
We would like to make the contribution process as smooth as possible.
Therefore, we provide an out of the box development environment that can be used with Visual Studio Code.
Please refer to the main [development guide](./DEVELOPMENT.md) to learn how to use this feature or setup your own development environment. 
```

**Review by pan- — APPROVED** [review]

**Pull request merged**